src/main/scala/com/mozilla/telemetry/sinks/RawHttpSink.scala

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package com.mozilla.telemetry.sinks
+
+import com.mozilla.telemetry.streaming.sinks.HttpSink
+import scalaj.http.{Http, HttpRequest}
+
+import scala.annotation.tailrec
+import scala.util.control.NonFatal
+import scala.util.{Failure, Success, Try}
+
+// TODO: Remove and use generalized HttpSink instead
```

Comment:
I saw that https://github.com/mozilla/telemetry-streaming/pull/147 has an updated HttpSink so this RawHttpSink class wouldn't be needed after that is merged